reduce.js

```diff
@@ -7,4 +7,11 @@ var numbers = [1, 56, 2, 4, 1, 99, 3, 5];
 // Write something that takes the above array and returns the product of all the numbers
 // via multiplication
 
+const add = (acc,next) => {
+    return acc+next;
+}
+const sum = numbers.reduce(add)
+// Write something that takes the above array and returns the product of all the numbers
+// via multiplication
+console.log(sum)
```

Comment:
nice work setting up your `reduce()`! However, re-reading the instructions, you're meant to return the **product**, not the sum, of the numbers.